recipes/restbed/all/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, load, rm, save
+from conan.tools.build import check_min_cppstd
+import os
+import re
+
+required_conan_version = ">=1.52.0"
+
+
+class RestbedConan(ConanFile):
+    name = "restbed"
+    homepage = "https://github.com/Corvusoft/restbed"
+    description = "Corvusoft's Restbed framework brings asynchronous RESTful functionality to C++14 applications."
+    topics = ("restful", "server", "client", "json", "http", "ssl", "tls")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "AGPL-3.0-or-later", "CPL"  # Corvusoft Permissive License (CPL)
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "ipc": [True, False],
+        "with_openssl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "ipc": False,
+        "with_openssl": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+        if self.settings.os in ("Windows", ):
+            del self.options.ipc
+
+    def validate(self):
+        if getattr(self.info.settings.compiler, "cppstd"):
+            check_min_cppstd(self, 14)
```

Comment:
Please add other compiler(clang, msvc, apple-clang) version checks like template:
https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L90-L96

Reply:
Addressed in 8e47eb107